nodes/src/nodes/tool_git/git_repo.py

```
@@ -0,0 +1,952 @@
+        repo = self._require_repo()
+        try:
+            b = repo.branches[branch]
+        except KeyError:
+            raise GitError(f'Branch {branch!r} not found') from None
+        repo.checkout(b)
+        return {
+            'branch': branch,
+            'sha': str(repo.head.target)[:8],
+        }
+
+    def branch_delete(self, name: str, force: bool = False) -> Dict[str, Any]:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)